db/schema.rb

```diff
@@ -794,6 +794,7 @@
     t.text "body_markdown"
     t.datetime "created_at", null: false
     t.string "description"
+    t.string "feature_flag"
```

Comment:
Rather than add a column here, it would be nice to keep the feature flags contained to just Flipper, why not use the slug as the flag instead? The slug will ensure uniqueness and is very unlikely to change for the life of the page.

Reply:
Feature flagging a page is going to be optional: we don't want to ever _require_ a feature flag: they are meant to be short lived. I'm going to use the slug name as the flag name as @citizen428 suggested  (https://github.com/thepracticaldev/dev.to/pull/8149#discussion_r432276233), but will still need to record the fact this page is guarded SOMEWHERE.